GitUI/Commands/GitExtensionCommandFactory.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace GitUI.Commands
+{
+    public interface IGitExtensionCommandFactory
+    {
+        IGitExtensionCommand Create();
+    }
+
+    public sealed class GitExtensionCommandFactory : IGitExtensionCommandFactory
+    {
+        private const string AboutCommandName = "about";
+
+        private readonly string[] _arguments;
+
+        public GitExtensionCommandFactory(string[] arguments)
+        {
+            _arguments = arguments;
+        }
+
+        public IGitExtensionCommand Create()
+        {
+            if (_arguments.Length <= 1)
+            {
+                return null;
```

Comment:
In the current setup, this should be handling Browse
(a comment would be sufficient at this stage)

Reply:
In Program.cs.

```
// until we complete the migration
if (сommand is null)
```

Reply:
Yes, but starting without parameters is not doing anything at all now, this comment is to remove the check when everything is complete.
You can add the comment there instead if you prefer or keep the args check for now

Reply:
Thanks! I understood. I made these commands, but I filled in only one. So yes, it won't work.
If there are objections, I think there will be no point in continuing.